projects/plugins/boost/app/assets/src/js/pages/getting-started/getting-started.tsx

```diff
@@ -0,0 +1,87 @@
+import { useState } from 'react';
+import { Snackbar } from '@wordpress/components';
+import { initializeConnection, getUpgradeURL } from '$lib/stores/connection';
+import { recordBoostEvent } from '$lib/utils/analytics';
+import { navigate } from '$lib/utils/navigate';
+import { BoostPricingTable } from '$features/boost-pricing-table/boost-pricing-table';
+import ActivateLicense from '$features/activate-license/activate-license';
+import Footer from '$layout/footer/footer';
+import Header from '$layout/header/header';
+import styles from './getting-started.module.scss';
+
+type GettingStartedProps = {
+	userConnected: boolean;
+	pricing: ( typeof Jetpack_Boost )[ 'pricing' ];
+	isPremium: boolean;
+	domain: string;
+};
+
+const GettingStarted: React.FC< GettingStartedProps > = ( {
+	userConnected,
+	pricing,
+	isPremium,
+	domain,
+} ) => {
+	const [ selectedPlan, setSelectedPlan ] = useState< 'free' | 'premium' | false >( false );
+	const [ snackbarMessage, setSnackbarMessage ] = useState< string >( '' );
+
+	if ( false !== selectedPlan ) {
+		initialize( selectedPlan );
+	}
+
+	async function initialize( plan: 'free' | 'premium' ) {
```

Comment:
`initialize()` is called before it's defined

Reply:
Should be resolved with 36ec873f63149e8a2879a0ee5eff65515348a3b2.